### PR TITLE
[4.0] Add anchors to user menu

### PR DESCRIPTION
**Pull request by Quy**
### Summary of Changes
Add anchors to link directly to tabs in user menu.

### Testing Instructions
Click User Menu icon in the upper right corner.
Click on Edit Account / Accessibility Settings links.

The respective tab should be opened/active.

### CODE REVIEW

**Comment by brianteeman** [comment]
the anchor is added but the link still doesnt work

image

**Comment by Quy** [comment]
Correct as noted in the Testing Instructions.

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 55f5e1970d2cc6fd8b8f7669c972048de399d43f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27850">issues.joomla.org/tracker/joomla-cms/27850</a>.</sub>

**Comment by Quy** [comment]
To testers, these are the same anchors when managing a user and hovering over `Account Details` and `Accessibility Settings` tabs.

File: administrator/modules/mod_user/tmpl/default.php [hunk 1]
```diff
@@ -34,12 +34,12 @@
 	<div class="dropdown-menu dropdown-menu-right">
 		<div class="dropdown-header"><?php echo $user->name; ?></div>
 		<?php $uri   = Uri::getInstance(); ?>
-		<?php $route = 'index.php?option=com_users&task=user.edit&id=' . $user->id . '&return=' . base64_encode($uri); ?>
+		<?php $route = 'index.php?option=com_users&task=user.edit&id=' . $user->id . '#details&return=' . base64_encode($uri); ?>
```
Comment (SharkyKZ) [inline administrator/modules/mod_user/tmpl/default.php]: It has to be last. Otherwise `return` will be ignored.

**Comment by mateoAdi** [comment]
I have tested this item :red_circle: unsuccessfully on 55f5e1970d2cc6fd8b8f7669c972048de399d43f<br /><br />Points me to the "Basic Settings" tab<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27850">issues.joomla.org/tracker/joomla-cms/27850</a>.</sub>

**Comment by brianteeman** [comment]
@mateoAdi Not where does the link go to. This is just to see if the url is correctly **trying** to go to #details

**Comment by Quy** [comment]
Links will now go to their respective tabs per https://github.com/joomla/joomla-cms/commit/1cb34cf1feb94df93e07a63d260116e223672486. Please retest.

**Comment by SharkyKZ** [comment]
I have tested this item :white_check_mark: successfully on bdd1d38ad32cb76274ad56808877e0f254ed6fd2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27850">issues.joomla.org/tracker/joomla-cms/27850</a>.</sub>

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on bdd1d38ad32cb76274ad56808877e0f254ed6fd2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27850">issues.joomla.org/tracker/joomla-cms/27850</a>.</sub>

**Comment by infograf768** [comment]
rtc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27850">issues.joomla.org/tracker/joomla-cms/27850</a>.</sub>

**Pull request merged**